static/docs/commands-reference/get-url.md

```diff
@@ -0,0 +1,166 @@
+# get-url
+
+Download or copy file or directory from any supported URL (for example `s3://`,
+`ssh://`, and other protocols) or local directory to the local file system.
+
+> Like `dvc init`, this is one of the few commands that doesn't require an
+> existing DVC project to run.
+
+> See `dvc get` to download data from other DVC repositories (e.g. GitHub URLs).
+
+> Unlike `dvc import-url`, this command does not track the downloaded data
+> file(s) (does not create a DVC-file).
```

Comment:
too many notes I think. let's keep the summary simple, put some of them into description. Some of them (like dvc get reference I would embed into the summary).

Reply:
Agreed. Fixed in 4d236f8

Reply:
does not require DVC repo to be run, and thus do not track ....